portability-transfer/src/main/java/org/dataportabilityproject/transfer/JobPollingService.java

```diff
@@ -83,22 +84,26 @@ private void pollForUnassignedJob() throws IOException {
     // so we don't have to make the user start from scratch. Some options are to manage this key
     // pair within our hosting platform's key management system rather than generating here, or to
     // encrypt and store the private key on the client.
-    // Note: claimJob may fail if another transfer worker beat us to it. That's ok -- this transfer
+    // Note: tryToClaimJob may fail if another transfer worker beat us to it. That's ok -- this transfer
     // worker will keep polling until it can claim a job.
-    claimJob(jobId, keyPair);
-    logger.debug(
-        "Updated job {} to CREDS_ENCRYPTION_KEY_GENERATED, publicKey length: {}",
-        jobId,
-        publicKey.getEncoded().length);
+    boolean claim = tryToClaimJob(jobId, keyPair);
+    logger.debug("Claim: " + claim);
```

Comment:
Can you consolidate this log with the one below, and make it clearer so you don't have to look up what "claim: false" means? i.e. "successfully claimed blah" or "failed to claim blah"

Reply:
Not sure if it makes sense to log it at all, actually.  Should be clear from the other logs what happened.